app/client/components/helpCentre/helpCentreConfig.ts

```diff
@@ -1,7 +1,13 @@
+interface Link {
+  title: string;
+  link: string;
+  id: string;
```

Comment:
Can we move `id` to the top of this interface? So that we have our unique identifiers at the start of the type.